backstory/src/main/java/com/google/sps/perspective/PerspectiveValues.java

```diff
@@ -0,0 +1,59 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data.perspective;
+
+import au.com.origma.perspectiveapi.v1alpha1.models.AttributeType;
+import java.util.Map;
+
+/**
+ * A container to hold text and the scores generated by that text.
+ */
+public class PerspectiveValues {
+  /** the text that generated these scores */
+  private final String text;
+
+  /** a map with type-score key-value pairs from the analysis of the text */
+  private final Map<AttributeType, Float> analyses;
```

Comment:
What do you think about naming this `attributeTypeToScoreMap`? `analyses` does sound better, but I'm afraid readers will have a hard time figuring out the values mean.

Alternatively, if a map is not strictly needed, and a collection would suffice, you could consider storing this as an array of PerspectiveAnalysis objects for even greater clarity, where:

```
class PerspectiveAnalysis {
  AttributeType attributeType;
  float score;  // would allow use of primitive float
}
```

Reply:
I see from a later PR that the map helps access to specific scores, so the alternative proposed above probably doesn't apply.

Reply:
I think that the reader will still understand the attribute type aspect of it because that is the key value. Perhaps I could document this better e.g. ```/** map containing the analysis from the Perspective API as AttributeType keys mapped to their corresponding scores. */ ```. I'm definitely open to changing it but I feel like ```analyses``` emphasizes that this is the results of the Perspective API analysis more so than the alternate name. @ykjchen What do you think?

Reply:
When writing code devs may reference the documentation to know more about it, but when reading code, it saves time not to need to look it up. For example, if I see somewhere:

```
boolean validate(PerspectiveValues perspectiveValues) {
  for (Map.Entry<AttributeType, Float> analysis : perspectiveValues.analyses.entrySet())  {
    if (analysis.getValue() < 0) {
      return false;
    }
  }
  return true;
}
```

I would wonder why 0 is invalid, because I don't know what the value represents. I'd have to navigate to the class to read the documentation and time == money.

```
boolean validate(PerspectiveValues perspectiveValues) {
  for (Map.Entry<AttributeType, Float> attributeTypeToScoreEntry : perspectiveValues.attributeTypeToScoreMap.entrySet())  {
    if (attributeTypeToScoreEntry.getValue() < 0) {
      return false;
    }
  }
  return true;
}
```

If I see this, I know immediately the value is a score, and it's intuitive that only >0 scores are valid. No navigation to documentation needed!

I'll leave it up to you, but I'd recommend favoring verbosity to potentially save people time in the long run.

Reply:
After a few offline conversations, I'm going to go change the name to ``typesToScores``` and update accordingly.

Reply:
Done